smsdk/client.py

```diff
@@ -225,3 +225,17 @@ def get_machine_schema(self, machine_source, types=[], return_mtype=False, **kwa
                             f"Unknow stat schema identified :: machine_type {machine_source} - "
                             f"title_prefix :: {stat.get('display', {}).get('title_prefix', '')}")
         return fields
+    
+    def get_kpis(self, **kwargs):
```

Comment:
Please add docstrings for the public methods, so that users can discover usage using `help()`